contrib/segyconverter/convert_segy.py

```diff
@@ -0,0 +1,134 @@
+"""
+Utility Script to convert segy files to blocks of numpy arrays and save to individual npy files
+"""
+
+import os
+import timeit
+import argparse
+import numpy as np
+import utils.segyextract as segyextract
+import utils.normalize_cube as normalize_cube
+import json
+
+
+def _clip(output_dir, stddev_file, k, min_range, max_range, normalize):
+    """
+    Normalization step on all files in output_dir
+    :param str output_dir: Directory path of all npy files to normalize
+    :param str stddev_file: txt file containing standard deviation result
+    :param int k: number of standard deviation to be used in normalization
+    :param float min_range: minium range value
+    :param float max_range: maximum range value
+    """
+    txt_file = os.path.join(output_dir, stddev_file)
+    if not os.path.isfile(txt_file):
+        raise Exception("Std Deviation file could not be found")
+    with open(os.path.join(output_dir, txt_file), 'r') as f:
+        metadatastr = f.read()
+        
+    try:
+        metadata = json.loads(metadatastr)
+        stddev = float(metadata['stddev'])
+        mean = float(metadata['mean'])
+    except ValueError:
+        raise Exception('stddev value not valid: {}'.format(metadatastr))
+
+    npy_files = list(f for f in os.listdir(output_dir) if f.endswith('.npy'))
+    for local_filename in npy_files:
+        normalize_cube.clip_file(os.path.join(output_dir, local_filename), stddev, mean,
```

Comment:
As far as I could understand, only clipped arrays (when normalized=False) will be saved. Should not we saved any "processed" array?